### PR TITLE
OSCI-4011: add ET teiid query for related advisories

### PR DESCRIPTION
**Pull request by Andrei-Stepanov**
Signed-off-by: Andrei Stepanov <astepano@redhat.com>

### CODE REVIEW

**Comment by Andrei-Stepanov** [comment]
@mgrabovsky please reivew.
This is ready to be merged.
Old comments are at https://github.com/fedora-ci/ciboard-server/pull/9
Thank you.

File: src/services/db.ts [hunk 1]
```diff
@@ -198,7 +198,8 @@ class DBCollection<TSchema extends Document = Document> {
 
   async cfg_indexes(): Promise<void> {
     this.log('Configure indexes.');
-    const indexes_config = _.get(cfg.db.collections[this.cfg_entry], 'indexes');
+    const collection_config = cfg.db.collections[this.cfg_entry];
+    const indexes_config = 'indexes' in collection_config ? collection_config.indexes : undefined;
```
Comment (mgrabovsky) [inline src/services/db.ts:202]: Shouldn't this be enough?
```suggestion
    const indexes_config = collection_config['indexes'];
```
That should make `indexes_config === undefined` if `'indexes'` is not present.

Comment (Andrei-Stepanov) [reply src/services/db.ts:202]: Yes, you are right. Interesting, that it works right now. Before eslint wasn't happy.

**Review by mgrabovsky — APPROVED** [review]
Looks good to me.

**Pull request merged**